br/backup-and-restore-faq.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Backup & Restore FAQ
+summary: Learn about Frequently Asked Questions (FAQ) and the solutions of BR.
+category: FAQ
+---
+
+# Backup & Restore FAQ
+
+This document lists the Frequently Asked Questions (FAQ) and the solutions about Backup & Restore (BR).
+
+If the problems are not covered in this document and cannot be resolved, you can raise questions in the [AskTUG](http://asktug.com) community.
+
+## What should I do if the error message `could not read local://...:download sst failed` is returned during data restoration?
+
+When restoring data, each node must have access to **all** backup files (SST files). By default, if `local` storage is used, you cannot restore data because the backup files are scattered among different nodes. Therefore, you have to copy the backup file of each TiKV node to the other TiKV nodes.
+
+It is recommended to mount an NFS disk as a backup disk during backup. For details, see [Back up a single table to a network disk](/br/backup-and-restore-use-cases.md#back-up-a-single-table-to-a-network-disk-recommended).
+
+## How much does it affect the cluster during backup using BR?
+
+When you use the `oltp_read_only` scenario of `sysbench` to back up to a disk (make sure the backup disk and the service disk are different) at full rate, the cluster QPS is decreased by 15%-25%. The impact on the cluster depends on the table schema.
+
+To reduce the impact on the cluster, you can use the `--ratelimit` parameter to limit the backup rate.
+
+## Does BR back up system tables? During data restoration, do they raise conflict?
+
+The system libraries (`information_schema`, `performance_schema`, `mysql`) are filtered out during full backup. For more details, refer to the [Backup Principle](/br/backup-and-restore-tool.md#backup-principle).
+
+Because these system libraries do not exist in the backup files, no conflict occurs among system tables during data restoration.
+
+## What should I do to resolve the `Permission denied` error, even if I have tried to run BR using root in vain?
+
+You need to confirm whether TiKV has access to the backup directory. To back up data, confirm whether TiKV has write privilege. To restore data, confirm whether it has read privilege.
```

Comment:
Maybe use `permission` would be better here, since the problem is "*Permission* denied"...?

Reply:
Updated in 123cb2651521cef80d5662a07b92fc9e0775bcd9~